givenergy_modbus_async/model/ems.py

```diff
@@ -0,0 +1,207 @@
+from enum import IntEnum, StrEnum
+import math
+
+from pydantic import BaseConfig, create_model
+
+from givenergy_modbus_async.model.register import HR, IR
+from givenergy_modbus_async.model.register import (
+    Converter as C,
+)
+from givenergy_modbus_async.model.register import (
+    RegisterDefinition as Def,
+)
+from givenergy_modbus_async.model.register import (
+    RegisterGetter,
+)
+
+class Generation(StrEnum):
+    """Known Generations"""
+
+    GEN1 = "Gen 1"
```

Comment:
seems to be a bit of duplication of these simple enums. Perhaps put them in a shared file like `__init__ `or something ?